extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js

```diff
@@ -138,9 +140,9 @@ require(
 			};
 			tracker.track(trackingParams);
 
-			cache.set(targetLanguage + 'WikiaInYourLangMessage' + cacheVersion, null);
+			cache.set(targetLanguage + 'WikiaInYourLangMessage' + articleTitle + cacheVersion, null);
 			// Cache for a month
-			cache.set('wikiaInYourLangNotificationShown' + cacheVersion, true, cache.CACHE_LONG);
+			cache.set('wikiaInYourLangNotificationShown' + articleTitle + cacheVersion, true, cache.CACHE_LONG);
```

Comment:
With this they'll have to dismiss a notification on every new article they view. Should they instead still just be able to dismiss once and not see a notification on any other article on that wikia for a month?